clusterloader2/pkg/prometheus/prometheus.go

```diff
@@ -335,6 +392,35 @@ func dumpAdditionalLogsOnPrometheusSetupFailure(k8sClient kubernetes.Interface)
 	klog.Info(string(s))
 }
 
+func getGceWindowsNodeNameFromKubernetesService(k8sClient kubernetes.Interface) (*gceNode, error) {
```

Comment:
Should this function and `type gceNode` go in a separate `gce.go` file?

Reply:
Ok, created `gce_windows_util.go` and put it there